### DISCUSSION TITLE
Depth vs Controlnet

### DISCUSSION THREAD

**Discussion by donbrasco01** [thygate/stable-diffusion-webui-depthmap-script#460] [General]
Hello people,

just out of curiosity, why is the generation of a depthmap so much faster with controlnet than the depthmap-script when using the same model?

**Comment by sonakravtsenko3** [comment]
Maybe different net size. What model do you use? Perhaps it is not exactly the same one.

**Reply by donbrasco01** [reply]
Hello semjon00,

i just figured out the depthmap from the script has a much higher resolution than the one im generating with controlnet. 

 It always used BOOST even the netsize-bars were visible. The generation without BOOST ist much faster.


Thanks for your reply!
